app/controllers/posts_controller.rb

```diff
@@ -5,6 +5,7 @@ def index
   end
 
   def show
+    @user = User.find(params[:user_id])
     @post = Post.find(params[:id])
   end
```

Comment:
- [ ] Good job adding the CanCanCan `ability` class Team. But, as required for the project and to correctly set up the user authorization using [CanCanCan](https://github.com/CanCanCommunity/cancancan), kindly make sure you use CanCanCan for Post authorization in Post Controller. 
> Something like...
```ruby  
class PostsController < ApplicationController
    load_and_authorize_resource

    ...
  end
```

Reply:
@TadesseDev  We appreciate your thoughtful feedback and we have made changes.